### DISCUSSION TITLE
Resize font

### DISCUSSION THREAD

**Discussion by droide** [6Morpheus6/pinokio-wiki#2] [General]
Hello,
how to resize font in Pinokio terminal?
Thanks.

**Comment by 6Morpheus6** [comment]
> Hello, how to resize font in Pinokio terminal? Thanks.

@droide
I don't think it is possible to resize only the font of the terminal.
I know there is a configuration file that defines the layout of Pinokio.
You can read about it here https://github.com/pinokiofactory/factory/discussions/20#discussioncomment-10687765
And below this post you can read about how to set the zoom level.
But if there is a way to resize the font in the terminal, you would find it in this configuration file.
If you figure it out, please let me know as well.
